gradle.properties

```diff
@@ -1,4 +1,4 @@
-project_version=4.1.5
+project_version=4.1.6
```

Comment:
the README says 4.2.0 but this is 4.1.6.  Which one is intended?

Reply:
Thanks, I pushed an update to make the correctrion